spec/datetime.html

```diff
@@ -552,6 +552,33 @@ <h1>Temporal.DateTime.prototype.getTime ( )</h1>
   <emu-clause id="sec-temporal-datetime-abstract-ops">
     <h1>Abstract operations</h1>
 
+    <emu-clause id="sec-temporal-getepochfromparts" aoid="GetEpochFromParts">
+      <h1>GetEpochFromParts ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_ )</h1>
+      <emu-alg>
+        1. Assert: _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, and _nanosecond_ are integer Number values.
+        1. Assert: _month_ ≥ 1 and _month_ ≤ 12.
+        1. Assert: _day_ ≥ 1 and _day_ ≤ ! DaysInMonth(_year_, _month_).
+        1. Assert: ! ValidateTime(_hour_, _minute_, _second_, _millisecond_) is *true*.
+        1. Let _date_ be ! MakeDay(_year_, _month_, _day_).
+        1. Let _time_ be ! MakeTime(_hour_, _minute_, _second_, _millisecond_).
+        1. Let _ms_ be ! MakeDate(_date_, _time_).
+        1. Assert: _ms_ is finite.
+        1. Return _ms_ × 1,000,000 + _microsecond_ × 1,000 + _nanosecond_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-datetimewithinlimits" aoid="DateTimeWithinLimits">
+      <h1>DateTimeWithinLimits ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_ )</h1>
+      <emu-alg>
+        1. Let _ns_ be ! GetEpochFromParts(_year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_).
+        1. If _ns_ ≤ -8.64 × 10<sup>21</sup><sub>ℝ</sub> - 8.64 × 10<sup>16</sup><sub>ℝ</sub>, then
```

Comment:
These numbers probably deserve an `<emu-note>` to explain where they come from.

Also, consider defining a constant nsPerDay as in https://tc39.es/ecma262/#sec-hours-minutes-second-and-milliseconds and calling it `100,000,001 &times; nsPerDay`